easybuild/easyconfigs/m/MaterialsStudio/MaterialsStudio-2022.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'Binary'
+
+name = 'MaterialsStudio'
+version = '2022'
+
+homepage = 'https://www.3dsbiovia.com/products/collaborative-science/biovia-materials-studio/'
+description = """Materials Studio is a complete modeling and simulation environment designed to allow researchers in
+materials science and chemistry to predict and understand the relationships of a material’s atomic and molecular
+structure with its properties and behavior. Using Materials Studio, researchers in many industries are engineering
+better performing materials of all types, including pharmaceuticals, catalysts, polymers and composites, metals and
+alloys, batteries and fuel cells, and more."""
+
+toolchain = SYSTEM
+
+sources = ['BIOVIA_%(version)s.MaterialsStudio%(version)s.tar']
```

Comment:
since the source can't be auto-downloaded, please add some `download_instructions`

Reply:
I actually don't know, a colleague manages the license and passed me the file, but I guess 

```python
download_instructions = """MaterialsStudio is proprietary software,
licensed users should be able to download it from https://www.3ds.com/support/download/"""
```

(?)

Reply:
Done